docs/content/guides/6.best-practices/3.ai-contexts.md

```diff
@@ -0,0 +1,69 @@
+# Coding with AI
+
+Alokai aims to boost developer productivity in many ways. E.g. by giving you storefront accelerator and OOTB integrations with 3rd party services.
+Now we want to go even further by boosting your productivity with AI.
+
+## Disclaimer
+
+AI field is evolving rapidly. New models, new tools, new workflows are emerging every day.
+While we do our best to keep this guide updated, we might miss some things.
+Revisit this guide regularly and don't follow it blindly.
+
+## AI Contexts
+
+Alokai provides a set of AI contexts that can be used to code with AI.
+//TODO: explain what are AI contexts
+
+## Expectations Management
+
+//TODO: elaborate on this points
+
+- AI is not a replacement for your knowledge and experience.
+- AI is a tool to help you code faster and more efficiently.
+- AI is not a replacement for your ability to reason and think.
+- AI is just predicts what would be the most likely outcome for the given input.
```

Comment:
You could just remove this line and the meaning would be the same.
I feel like it's intuitive to know that AI responds to input. What may be non-obvious is context that is being attached and effective communication. The "empathy" element that I was talking about. Just like you'd need to explain some things to another dev, you have to transfer this knowledge to the AI